liblauncher/elevation.cpp

```diff
@@ -0,0 +1,22 @@
+#include <Windows.h>
+#include <jni.h>
+
+extern "C" JNIEXPORT jboolean JNICALL Java_net_runelite_launcher_FilePermissionManager_isRunningElevated(JNIEnv *env, jclass clazz, jlong pid) {
+	BOOL fRet = false;
+	HANDLE hToken = nullptr;
+	HANDLE handle = OpenProcess(PROCESS_ALL_ACCESS, TRUE, pid);
```

Comment:
From googling I think you can use `PROCESS_QUERY_INFORMATION ` since you are only requesting the token?
```
PROCESS_QUERY_INFORMATION (0x0400) 	Required to retrieve certain information about a process, such as its token, exit code, and priority class (see [OpenProcessToken](https://learn.microsoft.com/en-us/windows/win32/api/processthreadsapi/nf-processthreadsapi-openprocesstoken)).
```

additionally there is no need for `bInheritHandle`